src/bin/vip-import-sql.js

```diff
@@ -506,6 +506,7 @@ Processing the SQL import for your environment...
 				environmentId: env.id,
 				basename: basename,
 				md5: md5,
+				searchReplace: [],
```

Comment:
Are this changes related?

Reply:
Not really, they just break flow validations.